theano/tensor/opt.py

```diff
@@ -63,54 +63,6 @@
 # Utilities
 
 
-def copy_stack_trace(from_var, to_var):
-    """
-    Copies the stack trace from one or more tensor variables to
-    one or more tensor variables.
-
-    :param from_var: tensor variable or list of tensor variables to
-                     copy stack traces from.
-    :param to_var: tensor variable or list of tensor variables to
-                     copy stack traces to.
-
-    .. note:: The stacktrace is assumed to be of the form of a list of lists
-    of tuples. Each tuple contains the filename, line number, function name
-    and so on. Each list of tuples contains the truples belonging to a
-    particular variable.
-    """
-
-    # Store stack traces from from_var
-    tr = []
-    if type(from_var) is list:
-        # If from_var is a list, store concatenated stack traces
-        if len(from_var) > 0:
-            for v in from_var:
-                if hasattr(v.tag, 'trace') and len(v.tag.trace) > 0:
-                    tr = tr + v.tag.trace
-    else:
-        # If from_var is not a list, it must be a single tensor
-        # variable, so just store that particular stack trace
-        if hasattr(from_var.tag, 'trace'):
-            tr = from_var.tag.trace
-
-    # Copy over stack traces to to_var
-    if type(to_var) is list:
-        # Copy over stack traces from from_var to each variable in
-        # to_var, including the stack_trace of the to_var before
-        for v in to_var:
-            if hasattr(v.tag, 'trace'):
-                v.tag.trace = v.tag.trace + tr
-            else:
-                v.tag.trace = tr
-    else:
-        # Copy over stack traces from from_var to each variable to
-        # to_var, including the stack_trace of the to_var before
-        if hasattr(to_var.tag, 'trace'):
-            to_var.tag.trace = to_var.tag.trace + tr
-        else:
-            to_var.tag.trace = tr
-
-
```

Comment:
You've merged the central/master of Theano into your branch. You shouldn't do this; if there is a conflict between your branch and the master, you should do a rebase of your branch over the master.